README.md

```diff
@@ -78,99 +78,64 @@ If you plan to fetch GitHub OWASP data locally, follow these additional steps:
 1. **Create a Super User**:
    - Run the following command to create a super user for accessing the admin interface:
 
-     ```bash
-     make setup
-     ```
+  ```bash
+  make setup
+  ```
 
 1. **Generate a GitHub Personal Access Token**:
-   - Create a GitHub [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+    - Create a GitHub [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
 
 1. **Update Environment Variables with GitHub Token**:
-   - Open `backend/.env/local` again and update it with your GitHub token:
+    - Open `backend/.env/local` again and update it with your GitHub token:
 
-     ```plaintext
-     GITHUB_TOKEN=<your_github_token>
-     ```
+      ```plaintext
+      GITHUB_TOKEN=<your_github_token>
+      ```
 
 1. **Sync Local Database Data**:
    - Now you should be able to run the following command to sync your local database data with GitHub:
 
-     ```bash
-     make sync
-     ```
-
-## Linting and Formatting Setup
-
-To ensure code quality, we utilize ESLint and Prettier with Husky for pre-commit hooks.
-
-### Installing Dependencies
-
-Make sure you have the necessary dependencies installed:
-
-```bash
-npm install --save-dev eslint prettier husky lint-staged eslint-plugin-react eslint-plugin-react-hooks @typescript-eslint/eslint-plugin @typescript-eslint/parser eslint-config-prettier eslint-plugin-prettier
-```
-
-### Configure Husky
+   ```bash
+   make sync
+   ```
 
-To set up Husky for pre-commit hooks, run:
+## Frontend Setup
```

Comment:
Just FYI: we want to avoid any local setup hassles. The dev env is dockerized for both FE/BE. I use `make run` to run the stack. 

No action required based on this comment.

Reply:
Got it ...thanks !